.github/workflows/goto-transcoder.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow executes the supported contracts in goto-transcoder
+
+name: Run GOTO Transcoder (ESBMC)
```

Comment:
We need to include a section in our book about the new tool. See Kani example https://model-checking.github.io/verify-rust-std/tools.html

Reply:
I added the documentation

Reply:
VeriFast added a page with an overview of the tool for users. Take a look at https://github.com/model-checking/verify-rust-std/pull/239. Could you include something similar? We already have one for [Kani](https://model-checking.github.io/verify-rust-std/tools/kani.html). It doesn't have to be super elaborate because it can grow over time, but we should give at least a overview for now.